ui/src/app/pages/data/concept/concept-homepage.tsx

```diff
@@ -359,6 +391,7 @@ export const ConceptHomepage = withCurrentWorkspace()(
       const {namespace, id} = this.props.workspace;
       this.setState({completedDomainSearches: [], concepts: [], countsError: false,
         domainErrors: [], countsLoading: true, searching: true});
+      currentConceptStore.next([]);
```

Comment:
I think this should happen only if `enableConceptSetSearchV2` is `true`. When I deploy with the flag disabled, I still see the shopping cart icon in the sidebar.

<img width="1303" alt="Screen Shot 2020-09-28 at 2 19 44 PM" src="https://user-images.githubusercontent.com/40036095/94477603-c0b97080-0197-11eb-9680-eb7fb10d8eed.png">